spring-cloud-gcp-storage/src/main/java/org/springframework/cloud/gcp/storage/GoogleStorageResource.java

```diff
@@ -48,72 +47,62 @@
  * @author Mike Eltsufin
  * @author Chengyuan Zhao
  * @author João André Martins
+ * @author Daniel Zou
  */
 public class GoogleStorageResource implements WritableResource {
```

Comment:
Can you just extend `GoogleStorageLocation`?

Reply:
Done.